lib/player.rb

```diff
@@ -0,0 +1,14 @@
+class Player
+  attr_reader :name, :deck
+  def initialize(name, deck)
+    @name = name
+    @deck = deck
+  end
+  def has_lost?
+    if @deck.cards.empty? == true
+      true
+    else
+      false
+    end
```

Comment:
this works and is very easy to read. You could also consider something like
```
def has_lost?
  @deck.cards.empty? # this will evaluate to either true or false
end
```